common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.common.diagnostics
+
+import com.revenuecat.purchases.common.Dispatcher
+import com.revenuecat.purchases.common.verboseLog
+import java.io.IOException
+
+class DiagnosticsTracker(
```

Comment:
It would help to have a docstring here too, otherwise I'd be wondering what the difference between this and `DiagnosticsManager` is.